### DISCUSSION TITLE
[Feature] Show assigned locations when multiple images are changed

### DISCUSSION THREAD

**Discussion by diabolo86** [immich-app/immich#13979] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sadly I'm not able/allowed to open an request.
I couldn't find any similar request or discussion.

If the location of multiple images is changed at once, it is not shown whether images already have a location in the metadata.
A good way to do this would be to display the assigned locations on the map that opens.
In addition, a button with "change all" or "change only empty locations" would be desirable.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Is this a duplicate of #12640?

**Reply by diabolo86** [reply]
Oh, sorry.
This one I hadn't seen.
This is kind of the same except the idea of showing the saved locations in the maps.
